easybuild/easyblocks/p/pgi.py

```diff
@@ -58,16 +61,88 @@ def extra_options():
     def __init__(self, *args, **kwargs):
         """Easyblock constructor, define custom class variables specific to PGI."""
         super(EB_PGI, self).__init__(*args, **kwargs)
-        if not self.cfg['license_file']:
-            self.cfg['license_file'] = 'UNKNOWN'
+
+        self.license_file = 'UNKNOWN'
+        self.license_env_var = 'UNKNOWN' # Probably not really necessary for PGI
+
         self.install_subdir = os.path.join('linux86-64', self.version)
 
     def configure_step(self):
         """
-        Dummy configure method, just a license check
+        Handle license file. It mimics intelbase.py behaviour. It should be merged into the framekwork
+        at some point
```

Comment:
I haven't tweak it much. Just removed the intel bits. You are absolutely right, it is basically a copy&paste. The reason for it is to make easier the transition from this to the new state with this functionality present in the framework. Meaning that all this should become a couple of lines in the future. I guess the options are:

1. Stick to a buggy easyblock until the new piece of the framework gets done and fix it then.
2. Accept a copy&paste to fix the easyblock ASAP and have a trivial transition once the new piece of the framework is done.

1 is more elegant but keeps things unsolved for an undefined amount of time. 2 is a quick and ugly hack, but that works well until we have what we need in the framework.

I'd suggest to go for 1 if the changes in the framework are done before the next release, or go for 2 otherwise.